sample/people.yaml

```diff
@@ -0,0 +1,15 @@
+- id: artem.zinnatullin
+  name: Artem Zinnatullin
+  twitter: artem_zin
+  github: artem-zinnatullin
+  website: "http://artemzin.com"
```

Comment:
https!

Reply:
The current feed includes HTTP then there and there 😨